franka_impedance_controller:
    type: cooperative_manipulation_controllers/FrankaCartesianImpedanceController
    arm_id: panda
    joint_names:
        - panda_joint1
        - panda_joint2
        - panda_joint3
        - panda_joint4
        - panda_joint5
        - panda_joint6
        - panda_joint7
    k_gains:
        - 200.0
        - 200.0
        - 200.0
        - 200.0
        - 200.0
        - 200.0
    d_gains:
        - 50.0
        - 50.0
        - 50.0
        - 20.0
        - 20.0
        - 20.0
    publish_rate: 10.0
    coriolis_factor: 1.0